src/Microsoft.Health.Fhir.Core/Features/Persistence/ResourceDeserializer.cs

```diff
@@ -49,5 +58,17 @@ internal ResourceElement DeserializeRaw(RawResource rawResource, string version,
 
             return deserializer(rawResource.Data, version, lastModified);
         }
+
+        internal ResourceElement DeserializeRaw(RawResourceElement rawResourceElement)
```

Comment:
It's not immediately obvious what the difference is between this and `Deserialize()`. Both have the same signature :)